<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>941</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Accueil</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>USER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>180</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>PASSEWORD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelogin">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lepassword">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Matriel</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1031</width>
        <height>771</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter </string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>971</width>
          <height>541</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Ajouter Matriel</string>
        </property>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>150</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>MISE_FONCTION</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>70</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="reference">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>170</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="mail">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>360</y>
           <width>231</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>60</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>MISE_FABRICATION</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>230</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>MARQUE</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>300</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ETAT</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>120</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>NOM</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>360</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>MAIL_FOURNISSEUR</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="marque">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>220</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nom">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>130</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>170</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>REFERENCE</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="etat">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>290</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonAjouter">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>340</y>
           <width>221</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="miseFab">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>90</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="miseFonc">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>190</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher </string>
       </attribute>
       <widget class="QTableView" name="tab_mat">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>1101</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>320</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID Matériel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_supp">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>320</y>
          <width>321</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="supp">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>320</y>
          <width>251</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>370</y>
          <width>251</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>291</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>420</y>
          <width>251</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Affecter à reparer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Mailling</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>230</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>70</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>170</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>320</y>
        <width>381</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dest name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dest mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Titre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>320</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Contenue</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
